README.md

```diff
@@ -920,6 +1160,43 @@ On the destination, after all devices have been migrated, you should see logs li
 2024/06/24 13:55:16 Resumed VM in 60.026997ms on out/vms/firecracker/39zY39Y9Gs8N4wPMs5WoLe/root
 ```
 
+If you're not happy with migration performance, encounter resource contention issues, or want to optimize for something other than minimal downtime, here are configuration options for `drafter-peer`:
+
+- `--disable-postcopy-migration`: Prevents the instance from resuming on the destination before all data is locally available. This reduces total migration time but increases downtime.
+
+- `--concurrency`: Sets concurrent workers for migrations (default 1024). Lower values reduce CPU usage but may increase migration time and downtime, especially in high-latency environments.
```

Comment:
Worth noting here that higher concurrency will use memory... specifically migration buffer usage will be `concurrency` * `block_size` - eg with 1m block size, and 1024 concurrency, memory usage will be 1gb+

Reply:
Fixed in 055f335b33d1f5df1ff478b8066b84d7136b6e3b